fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -87,6 +87,19 @@ object FAPI {
     for(snapContent <- ContentApi.latestContentFromLatestSnaps(capiClient, allSnaps, adjustItemQuery))
       yield snapContent}
 
+  def getTreatsForCollection(collection: Collection, adjustSearchQuery: AdjustSearchQuery = identity, adjustItemQuery: AdjustItemQuery = identity)
```

Comment:
The formatting's a bit all over the place in a couple of places here. The arg list and the for comp jump out?